src/main/java/tictactoe/ComputerPlayer.java

```diff
@@ -94,7 +94,7 @@ private int maximizingPlayer(Board board, MarksEnum currentMark, int alpha, int
         if (--depth == 0) {
             return Result.WIN.value;
         }
-        for (Integer position : board.freePosition()) {
+        for (Integer position : board.freePositions()) {
```

Comment:
Can you just use `int position` here? I think it should auto-unbox to a primitive.